src/test/fuzz/blockfilter.cpp

```diff
@@ -15,22 +15,26 @@
 FUZZ_TARGET(blockfilter)
 {
     FuzzedDataProvider fuzzed_data_provider(buffer.data(), buffer.size());
-    const std::optional<BlockFilter> block_filter = ConsumeDeserializable<BlockFilter>(fuzzed_data_provider);
-    if (!block_filter) {
+    const std::optional<uint256> u256 = ConsumeDeserializable<uint256>(fuzzed_data_provider);
+    if (!u256) {
         return;
     }
```

Comment:
I'm not too deep into fuzzing, but IIUC, we just want a random uint256 here and there is not really any benefit by involving a deserialization (that could fail)?
```suggestion
    const uint256 u256 = ConsumeUInt256(fuzzed_data_provider);
```
Could also limit this variable's scope by moving it down before it's first use (that is, before GCSFilter is constructed).

Reply:
> we just want a random uint256 here and there is not really any benefit by involving a deserialization (that could fail)?

Yeah that's right. I'll apply the suggested change.